turbo.json

```diff
@@ -24,6 +24,10 @@
       "dependsOn": ["build"],
       "outputs": []
     },
+    "profile": {
+      "dependsOn": ["@quri/squiggle-lang#build"],
```

Comment:
Since squiggle-lang doesn't depend on any other packages in this repo, depending on generic `build` is enough and more robust (for example, if `components` will ever get a `profile` target, it should probably depend on components `build` stage).